docs/articles_en/get_started/installing-openvino-overview/installing-openvino-windows-header/installing-openvino-from-archive-windows.rst

```diff
@@ -240,11 +240,9 @@ Additional Resources
 
 <!---
    To learn more about converting models from specific frameworks, go to:
-   * :ref:`Convert Your Caffe Model <convert model caffe>`
```

Comment:
maybe we should remove this whole piece - it is commented out anyway and the html comment formatting fails to render properly with current builds

Reply:
Change applied in https://github.com/openvinotoolkit/openvino/pull/22509/commits/ef18601b07d035c40555cb80978cca7d3fb76669